docs/core/windows-prerequisites.md

```diff
@@ -13,45 +13,117 @@ ms.assetid: c33b1241-ab66-4583-9eba-52cf51146f5a
+
+There are different editions of Visual Studio 2017. You can download [Visual Studio Community 2017](https://www.visualstudio.com/downloads/) for free to get started.  
 
-There are different editions of Visual Studio 2017. You can download [Visual Studio Community 2017](https://www.visualstudio.com/downloads/) for free to get started.  To learn more about the Visual Studio installation process, see [Install Visual Studio 2017](/visualstudio/install/install-visual-studio).
+To learn more about the Visual Studio installation process, see [Install Visual Studio 2017](/visualstudio/install/install-visual-studio).
 
 To verify that you're running the latest version of Visual Studio 2017, do the following:
 
  * On the **Help** menu, choose **About Microsoft Visual Studio**.
  * In the **About Microsoft Visual Studio** dialog, the version number should be 15.0.26228.4 or higher.
 
 You can read more about the changes in Visual Studio 2017 in the [release notes](https://www.visualstudio.com/news/releasenotes/vs2017-relnotes).
+
+[.NET Core 1.x](#tab/netcore1x)
+
+> [!IMPORTANT]
+> Even though, it's possible to use Visual Studio 2015 with a preview version of the .NET Core tooling, these projects are *project.json*-based, which is now deprecated. Visual Studio 2017 uses project files based on MSBuild. For more information about the format changes, see [High-level overview of changes](./tools/cli-msbuild-architecture.md).
+
+To develop .NET Core 1.x apps in Visual Studio, install the [latest version of Visual Studio](https://www.visualstudio.com/downloads/) with the **.NET Core cross-platform development** toolset (in the **Other Toolsets** section) selected.
+
+![Screenshot of Visual Studio 2017 installation with the ".NET Core cross-platform development" workload selected](./media/windows-prerequisites/vs_workloads.jpg)
+
+     To verify that you're running the latest version of Visual Studio 2017, do the following:
+
+         * On the **Help** menu, choose **About Microsoft Visual Studio**.
+         * In the **About Microsoft Visual Studio** dialog, the version number should be 15.0.26228.4 or higher.
+
+
+[.NET Core 2.x](#tab/netcore2x)
+
+Visual Studio 2017 continues to use .NET Core 1.x by default. Install the .NET Core 2.0 SDK to get .NET Core 2.0 support in Visual Studio 2017.
+
+Once installed, Visual Studio 2017 uses the .NET Core SDK 2.0, and supports all of the following actions:
+
+  * Open, build, and run your existing .NET Core 1.x projects.
+  * Retarget your .NET Core 1.x projects to 2.0 and then build and run on .NET Core 2.0.
+  * Create new .NET Core 2.0 projects
+
+To develop .NET Core 2.0 apps in Visual Studio 2017:
+
+ 1. Install the [.NET Core 2.0 SDK](https://www.microsoft.com/net/download/core) .
+ 2. Install the [latest version of Visual Studio](https://www.visualstudio.com/downloads/) with the **.NET Core cross-platform development** toolset (in the **Other Toolsets** section) selected.
```

Comment:
The top tab on the installer says "Workloads", but the section says "Other Toolsets". That was already in the article.